src/constants/editors.ts

```diff
@@ -9,12 +18,12 @@ export const VSCODE = "VSCODE";
 export const XCODE = "XCODE";
 
 export default [
-  ATOM,
-  CHROME,
-  ECLIPSE,
-  ANDROIDSTUDIO,
-  SUBLIMETEXT2,
-  SUBLIMETEXT3,
-  VIM,
-  VSCODE
+  { name: ATOM, instance: new Atom() },
```

Comment:
Each editor class has it's own `name` attribute. Let's just `export default [Atom, SublimeText2...` then instantiate each class in the local scope where they're used.